website/docs/configuration/expressions.html.md

```diff
@@ -13,6 +13,8 @@ description: |-
 earlier, see
 [0.11 Configuration Language: Interpolation Syntax](../configuration-0-11/interpolation.html).
 
+> **Hands-on:** Try the [Create Dynamic Expressions](https://learn.hashicorp.com/tutorials/terraform/expressions?in=terraform/configuration-language&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```

Comment:
If you use `->` as a prefix here, rather than `>`, then the custom Markdown extension we have in the website renderer will present it as a blue "information" callout box rather than as a blockquote, which I think would make it a little more prominent on the page, assuming that prominence is desirable. :grinning: 

(With that said, in this particular case I suppose there is _already_ a blue callout box right above this, so maybe that would just look _weird_... :confused: )

Reply:
Hmm.. I was going with the formatting from https://www.terraform.io/docs/configuration/variables.html but it does indeed look weird

Reply:
Yeah, we went back and forth on this when I was originally doing these "hands-on" callouts, and Judith and I settled on (mis)using the blockquote syntax to set it apart in a less obtrusive way. They tend to go near-ish to a blue box, so we didn't want to create too much blue box chaos. 

tl;dr plain `>` is correct in this case.